src/command/client/search/interactive.rs

```diff
@@ -289,13 +299,28 @@ impl State {
         let input = Paragraph::new(input);
         f.render_widget(input, chunks[2]);
 
+        let selected = self.results_state.selected();
+        let command = if results.is_empty() {
+            ""
+        } else {
+            &results[selected].command
+        }
+        .chars()
+        .collect::<Vec<char>>()
+        .chunks((f.size().width - 2).into())
+        .map(|chunk| chunk.iter().collect::<String>())
+        .collect::<Vec<String>>()
+        .join("\n");
+        let preview = command;
+        let preview = Paragraph::new(preview).style(Style::default().fg(Color::DarkGray));
+        f.render_widget(preview, chunks[3]);
+
         let extra_width = UnicodeWidthStr::width(self.input.substring());
 
         f.set_cursor(
             // Put cursor past the end of the input text
             chunks[2].x + extra_width as u16 + PREFIX_LENGTH + 1,
-            // Move one line down, from the border to the input line
-            chunks[2].y + 1,
+            chunks[2].y,
```

Comment:
I've noticed this as soon as I added the preview chunk below the input as the cursor was place one line below the input text.